trino/trino.go

```diff
@@ -2185,6 +2200,169 @@ func (s *NullSlice3Map) Scan(value interface{}) error {
 	return nil
 }
 
+func scanNullSliceByte(v interface{}) (NullSliceByte, error) {
+	if v == nil {
+		return NullSliceByte{}, nil
+	}
+
+	sliceByte := NullSliceByte{}
+	err := sliceByte.Scan(v)
+	if err != nil {
+		return NullSliceByte{}, err
+	}
+	return sliceByte, nil
+}
+
+// NullSliceByte represents a []byte that can be null.
+// The NullSliceByte supports Trino's Varbinary type.
+type NullSliceByte struct {
+	SliceByte []byte
+	Valid     bool
+}
+
+// Scan implements the sql.Scanner interface.
+func (b *NullSliceByte) Scan(value interface{}) error {
+	if value == nil {
+		b.SliceByte, b.Valid = []byte{}, false
+		return nil
+	}
+	b.Valid = true
+
+	var err error
+	switch v := value.(type) {
```

Comment:
Can the server really return `[]byte` here or is it always a base64 encoded string? If the latter, remove the switch statement.